frontend/src/metabase/static-viz/components/LineAreaBarChart/utils/series.tsx

```diff
@@ -74,10 +77,6 @@ export function getSeriesWithLegends(
         if (!hasTwoDimensions(series)) {
           // One or zero dimensions
 
-          if (seriesIndex === 0 && series.name) {
-            return series.name;
-          }
```

Comment:
We don't really use `name` which initially was meant for custom column and metric name, but turns out that since we calculate series with 2 dimensions' key on FE (breakout), we should do the same for series with 1 dimension for consistency, that's why this is removed. The BE change can be seen here.

https://github.com/metabase/metabase/pull/26797/files#diff-79c051a13f9e859c56dc5bffd3f145bb696e2da951d6ff45dc1fa38d4a3b9807L656